### ISSUE TITLE
urbit-king crashes, KILLING: processWork

### ISSUE THREAD

**Issue by aquarial** [urbit/urbit#5441]
**Describe the bug**

Having trouble with 1.7 urbit-king. System has enough memory/swap etc. 

```bash
~/urbit$ ./urbit -F zod
~
urbit 1.7
boot: home is ~/zod
... etc
lite: final state 2f90a2ff
~zod:dojo> C-d

~/urbit$ ./urbit-king run zod
loom: mapped 2048MB
boot: protected loom
live: loaded: MB/748.158.976
boot: installed 348 jets
ship is ~zod
"KILLING: processWork"

~/urbit$     # urbit-king crashed
```

**System (please supply the following information, if relevant):**
 - OS: linux64 ubuntu

**Additional context**

Using the most recent release binaries for linux64:

```
~/urbit$ sha256sum urbit*
b0ed46e90cfde50f16e46b92fbe675765eef9d147b62542697653cb44f50c01d  urbit
b34c5e0fc58672888e8c07cd3e0fd232f31a87dd7f9e71ff9a7183a5bd32e856  urbit-king
6a197329fddd39ed604620b9179b72ec1bbf3f5d8797357e817087d0a7be34f4  urbit-worker
```

The error is printed in IPC.hs: https://github.com/urbit/urbit/blob/28cc660998e6dff562f052cad271082c431e2112/pkg/hs/urbit-king/lib/Urbit/Vere/Serf/IPC.hs#L596-L602

**Comment by aquarial** [comment]
Also having this issue when I use `urbit-king` for my real ship + moons. `./urbit` binary works fine, but `./urbit-king` crashes with the error message.

I prefer urbit-king because I like to move the http port from 8080.

**Comment by aquarial** [comment]
Messaging from The Forge:

> urbit-king was accidentally not flagged as compatible with zuse 419
> will have to use C urbit and wait for v1.8

> The King is deprecated

Long live the King.

**Comment by aquarial** [comment]
Commit https://github.com/urbit/urbit/commit/1c26ef6fe90b469598f9c5e1a52757ea9e9f6812 released in 1.8 fixed this, can't reproduce the crash.
